style.css

```diff
@@ -1559,4 +1808,302 @@ h2 {
     position: absolute;
     left: 45%;
   }
+
+  .model {
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+    position: fixed;
+    top: 50%;
+    left: 50%;
+    padding: 0 20px;
+    transform: translate(-50%, -50%) scale(0);
+    transition: 200ms ease-in-out;
+    z-index: 10;
+    background: #fff;
+    border: 1px solid #ebecf0;
+    box-shadow: 0px 48px 48px rgba(37, 47, 137, 0.08);
+    border-radius: 16px;
+    width: 1156px;
+    max-width: 70%;
+  }
+
+  .model.active {
+    transform: translate(-50%, -50%) scale(1);
+  }
+
+  #overlay {
+    position: fixed;
+    opacity: 0;
+    transition: 200ms ease-in-out;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background: #413c3b;
+    mix-blend-mode: multiply;
+    pointer-events: none;
+  }
+
+  #overlay.active {
+    opacity: 1;
+    pointer-events: all;
+  }
+
+  .popupContainer {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .h2popup {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    text-align: left;
+    margin-left: 0;
+    width: 156px;
+    height: 40px;
+  }
+
+  .close {
+    cursor: pointer;
+    border: none;
+    outline: none;
+    background: none;
+    color: #67798e;
+    font-size: 30px;
+    margin-left: 780px;
+  }
+
+  .model .popuptonic {
+    display: flex;
+    gap: 8px;
+    margin-top: 15px;
+    padding: 0;
+    align-items: center;
+    /* width: 260px;
+    height: 36px; */
+  }
+
+  .popuptonic.canopy {
+    width: 63px;
+    height: 20px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  .popuptonic.back {
+    width: 101px;
+    height: 20px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 20px;
+    color: #7a869a;
+  }
+
+  .popuptonic.dot {
+    margin-top: 5;
+    width: 8px;
+    height: 8px;
+    background: #c1c7d0;
+    border-radius: 50%;
+  }
+
+  .popuptonic.year {
+    width: 32px;
+    height: 20px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 20px;
+    color: #7a869a;
+  }
+
+  .model-img {
+    width: 100%;
+    height: 350px;
+  }
```

Comment:
Great job team 🤝

- [ ] Kindly make sure your popup is responsive to all screen sizes. Some features of your design seem to be overflowing as the screen size reduces.
![mobile-popup](https://user-images.githubusercontent.com/63917725/211171969-72d0b3ff-b922-460a-b401-4c83d684998b.PNG)

- [ ] You may also make sure the image of the desktop popup is a close replica of the one given in the Figma template. Your image appears to be stretched and blurred.
![desktop-popup](https://user-images.githubusercontent.com/63917725/211172055-beaa26f2-2a85-493f-94e9-a13efd1b8d8c.PNG)